src/fromager/external_commands.py

```diff
@@ -37,7 +37,8 @@ def detect_network_isolation():
     """
     cmd = network_isolation_cmd()
     if os.name == "posix":
-        subprocess.check_call(cmd + ["true"], stderr=subprocess.STDOUT)
+        check = [*cmd, "true"]
+        subprocess.check_call(check, stderr=subprocess.STDOUT)
```

Comment:
I think this all needs to be called using `external_commands.run`. I will add it to the existing issue

Reply:
The function runs before we have logging framework set up.

Reply:
Ah got it